llama_finetuning.py

```diff
@@ -12,21 +12,23 @@
 from torch.distributed.fsdp import (
     FullyShardedDataParallel as FSDP,
 )
+from torch.distributed.fsdp.fully_sharded_data_parallel import CPUOffload
 from torch.optim.lr_scheduler import StepLR
 from torch.utils.data import DistributedSampler
 from transformers import (
     LlamaForCausalLM,
-    LlamaTokenizer,
+    AutoTokenizer,
```

Comment:
Why do we change this to AutoTokenizer?

Reply:
To support the CodeLLamaTokenizer automatically